swap_meet/vendor.py

```diff
@@ -1,2 +1,114 @@
+from .item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
```

Comment:
👍 This constructor allows an explicit inventory to be passed in, but if omitted, we will use a fresh empty list.

Reply:
Note that for default arguments, we _omit_ the spaces around `=` to help it appears visually distinct from regular assignment.

```py
    def __init__(self, inventory=None):
```